chain/jsonrpc/tests/rpc_query.rs

```diff
@@ -14,30 +14,13 @@ use near_primitives::test_utils::init_test_logger;
 use near_primitives::types::{BlockId, BlockIdOrFinality, Finality, ShardId};
 use near_primitives::views::{QueryRequest, QueryResponseKind};
 
+#[macro_use]
 mod test_utils;
 
-macro_rules! test_with_client {
-    ($client:ident, $block:expr) => {
-        init_test_logger();
-
-        System::run(|| {
-            let (_view_client_addr, addr) = test_utils::start_all(false);
-
-            let mut $client = new_client(&format!("http://{}", addr));
-
-            actix::spawn(async move {
-                $block.await;
-                System::current().stop();
-            });
-        })
-        .unwrap();
-    };
-}
-
 /// Retrieve blocks via json rpc
 #[test]
 fn test_block_by_id_height() {
-    test_with_client!(client, async move {
+    test_with_client!(test_utils::NodeType::NonValidator, client, async move {
```

Comment:
Why not `use test_utils::NodeType`?

Reply:
I just wanted to make sure it is clear that this is an ad-hoc enum, not something that is a part of the core types.